Tasks/AzureRmWebAppDeploymentV4/deploymentProvider/WindowsWebAppWarDeployProvider.ts

```diff
@@ -19,16 +19,21 @@ export class WindowsWebAppWarDeployProvider extends AzureRmWebAppDeploymentProvi
 
         this.zipDeploymentID = await this.kuduServiceUtility.deployUsingWarDeploy(this.taskParams.Package.getPath(), 
             { slotName: this.appService.getSlot() }, warName);
-
+       
         await this.PostDeploymentStep();
     }
-    
+
+
     public async UpdateDeploymentStatus(isDeploymentSuccess: boolean) {
-        if(this.kuduServiceUtility) {
+        if(!this.kuduServiceUtility){
+            tl.debug('Kudu service utility not found.');
```

Comment:
Instead of debug, maybe we should use error?

Reply:
Many translation changes are required. Let's skip it for now and we can consider it in future.